### DISCUSSION TITLE
Is there a way to get a mouse pointer's info from unity?

### DISCUSSION THREAD

**Discussion by DeokjinLee** [raisimTech/raisimLib#238] [Q&A] [answered]
Hello, raisimTech.

1. Is there a way to get a mouse pointer's information from Unity?
I want to make a robot arm follow my mouse pointer in Unity or input disturbance to certain point on a robot using my mouse.
I guess if I can find a mouse event handler, all question above would be solved.

2. How to draw a line in Unity from code?
For example, I would like to display robot's end-effector path in unity.

Thank you.

**Comment by jhwangbo** [comment, marked as answer]
1. No. Because raisimUnity is in a Server-Client setting, it is difficult to implement custom mouse inputs
2.  checkout "polyline" in the server

**Reply by DeokjinLee** [reply]
About Q1, it seems to use a mouse event in the simulation below link.
Could you let me know how you made it?

https://www.youtube.com/watch?v=RoTYz_cqK7w&t=3466s
26:43~27:35

Thank you.

**Reply by jhwangbo** [reply]
Unfortunately, that visualizer is too old and unusable at this point.
